README.md

```diff
@@ -51,17 +51,20 @@ interface TextChangedListener {
 }
 
 class PrintingTextChangedListener : TextChangedListener {
-
-    override fun onTextChanged(oldText: String, newText: String) =
-        println("Text is changed $oldText -> $newText")
+    
+    var text = ""
```

Comment:
This seam not used, can you remove it?

Reply:
It is used, in row 58:
```text = "Text is changed: $oldText -> $newText"``` 
Also later in the test, which is not included in the readme.